CMakeLists.txt

```diff
@@ -6,13 +6,33 @@
 #The full license is in the file LICENSE, distributed with this software. 
 #############################################################################
 
-cmake_minimum_required(VERSION 3.24)
+cmake_minimum_required(VERSION 3.5)
 project(xeus-r)
+
 set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake;${CMAKE_MODULE_PATH}")
 set(XEUS_R_INCLUDE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/include)
 set(CMAKE_VERBOSE_MAKEFILE ON)
 set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -Wl,-v")
```

Comment:
These linker flags (`-Wl,`) do not seem to be working so well with `em++`, maybe we should wrap them in another if/else block.

> ```/home/runner/micromamba/envs/xeus-r-wasm-build/lib/python3.13/site-packages/emsdk/upstream/emscripten/em++  -std=c++17 -O3 -DNDEBUG  -Wl,-v --bind -Wno-unused-command-line-argument -fexceptions -s MODULARIZE=1 -s EXPORT_NAME=createXeusModule -s EXPORT_ES6=0 -s USE_ES6_IMPORT_META=0 -s DEMANGLE_SUPPORT=0 -s ASSERTIONS=0 -s ALLOW_MEMORY_GROWTH=1 -s EXIT_RUNTIME=1 -s WASM=1 -s USE_PTHREADS=0 -s ENVIRONMENT=web,worker -s TOTAL_STACK=32mb -s INITIAL_MEMORY=128mb -s WASM_BIGINT -s FORCE_FILESYSTEM -s MAIN_MODULE=1 @CMakeFiles/xr.dir/objects1.rsp -o xr.js @CMakeFiles/xr.dir/linkLibs.rsp```

Reply:
Yes I was very doubtful about this. So I built a second xeus kernel against emscripten (xeus-cpp in this case) and that technically involved all of the above flags except the -Wl one. So I think we might do without it.